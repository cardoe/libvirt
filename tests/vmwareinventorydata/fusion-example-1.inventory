.encoding = "UTF-8"
vmlist1.config = "/Volumes/POO/Virtual Machines.localized/MS XP.vmwarevm/MS XP.vmx"
vmlist1.DisplayName = "MS XP"
vmlist1.ParentID = "0"
vmlist1.ItemID = "1"
vmlist1.SeqID = "0"
vmlist1.IsFavorite = "FALSE"
vmlist1.IsClone = "FALSE"
vmlist1.CfgVersion = "8"
vmlist1.State = "normal"
vmlist1.UUID = "56 4d 6e e1 b2 84 7f 71-07 d1 08 4a bb 61 1b f6"
vmlist4.config = "/Volumes/POO/Virtual Machines.localized/VMs/ATTM CentOS 64-bit.vmwarevm/ATTM CentOS 64-bit.vmx"
vmlist4.DisplayName = "ATTM CentOS 64-bit"
vmlist4.ParentID = "0"
vmlist4.ItemID = "4"
vmlist4.SeqID = "2"
vmlist4.IsFavorite = "FALSE"
vmlist4.IsClone = "FALSE"
vmlist4.CfgVersion = "8"
vmlist4.State = "normal"
vmlist4.UUID = "56 4d 49 4f 39 1c a8 ed-f0 95 12 41 5b 07 34 bb"
vmlist11.config = "/Volumes/POO/Virtual Machines.localized/Windows Vista.vmwarevm/Windows Vista.vmx"
vmlist11.DisplayName = "Windows Vista"
vmlist11.ParentID = "0"
vmlist11.ItemID = "11"
vmlist11.SeqID = "3"
vmlist11.IsFavorite = "FALSE"
vmlist11.IsClone = "FALSE"
vmlist11.CfgVersion = "8"
vmlist11.State = "normal"
vmlist11.UUID = "56 4d 4e e2 48 c0 c1 3a-97 f4 d0 de ad 0b 88 60"
vmlist18.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node01.vmwarevm/maas-node01.vmx"
vmlist18.DisplayName = "maas-node01-95 / qft7d"
vmlist18.ParentID = "23"
vmlist18.ItemID = "18"
vmlist18.SeqID = "1"
vmlist18.IsFavorite = "FALSE"
vmlist18.IsClone = "FALSE"
vmlist18.CfgVersion = "8"
vmlist18.State = "broken"
vmlist18.UUID = "56 4d 0b 2e c0 12 37 af-b0 e2 31 14 ab e4 8a 95"
vmlist20.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node03.vmwarevm/maas-node03.vmx"
vmlist20.DisplayName = "maas-node03-38 / 4jmj4"
vmlist20.ParentID = "23"
vmlist20.ItemID = "20"
vmlist20.SeqID = "2"
vmlist20.IsFavorite = "FALSE"
vmlist20.IsClone = "FALSE"
vmlist20.CfgVersion = "8"
vmlist20.State = "broken"
vmlist20.UUID = "56 4d a6 6f 10 99 e4 2a-12 99 c5 74 a7 d9 ed 38"
vmlist21.config = "/Volumes/POO/Virtual Machines.localized/VMs/nagiosxi-64/nagiosxi-64.vmx"
vmlist21.DisplayName = "nagiosxi-64"
vmlist21.ParentID = "0"
vmlist21.ItemID = "21"
vmlist21.SeqID = "5"
vmlist21.IsFavorite = "FALSE"
vmlist21.IsClone = "FALSE"
vmlist21.CfgVersion = "8"
vmlist21.State = "normal"
vmlist21.UUID = "56 4d f6 41 45 81 0d 17-07 6d 14 56 92 b7 50 2a"
vmlist22.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node04.vmwarevm/maas-node03.vmx"
vmlist22.DisplayName = "maas-node04-42 / ejhfe"
vmlist22.ParentID = "23"
vmlist22.ItemID = "22"
vmlist22.SeqID = "3"
vmlist22.IsFavorite = "FALSE"
vmlist22.IsClone = "FALSE"
vmlist22.CfgVersion = "8"
vmlist22.State = "broken"
vmlist22.UUID = "56 4d 41 e2 15 1e 12 3c-3d f1 21 b7 08 61 ac 42"
vmlist23.config = "folder23"
vmlist23.Type = "2"
vmlist23.DisplayName = "MAAS"
vmlist23.ParentID = "0"
vmlist23.ItemID = "23"
vmlist23.SeqID = "6"
vmlist23.IsFavorite = "FALSE"
vmlist23.UUID = "folder:52 3a d2 50 96 b3 cd 22-eb 2c 82 2f 9c 7b ab 72"
vmlist23.Expanded = "TRUE"
vmlist24.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node05.vmwarevm/maas-node03.vmx"
vmlist24.DisplayName = "maas-node05-AB / mhknt"
vmlist24.ParentID = "23"
vmlist24.ItemID = "24"
vmlist24.SeqID = "4"
vmlist24.IsFavorite = "FALSE"
vmlist24.IsClone = "FALSE"
vmlist24.CfgVersion = "8"
vmlist24.State = "broken"
vmlist24.UUID = "56 4d 01 e7 63 18 f2 80-5f a6 c4 0a fe e1 10 ab"
vmlist25.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node06.vmwarevm/maas-node03.vmx"
vmlist25.DisplayName = "maas-node06-3B-96 / r8g3j"
vmlist25.ParentID = "23"
vmlist25.ItemID = "25"
vmlist25.SeqID = "5"
vmlist25.IsFavorite = "FALSE"
vmlist25.IsClone = "FALSE"
vmlist25.CfgVersion = "8"
vmlist25.State = "broken"
vmlist25.UUID = "56 4d 83 39 96 d6 68 da-75 f8 9e 8a 7f f7 3b 96"
vmlist26.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node07.vmwarevm/maas-node03.vmx"
vmlist26.DisplayName = "maas-node07-69 / ptpjg"
vmlist26.ParentID = "23"
vmlist26.ItemID = "26"
vmlist26.SeqID = "6"
vmlist26.IsFavorite = "FALSE"
vmlist26.IsClone = "FALSE"
vmlist26.CfgVersion = "8"
vmlist26.State = "broken"
vmlist26.UUID = "56 4d ac 87 f5 52 c2 f6-4d 53 26 bc 60 03 b7 69"
vmlist28.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node08.vmwarevm/maas-node03.vmx"
vmlist28.DisplayName = "maas-node28-3A"
vmlist28.ParentID = "54"
vmlist28.ItemID = "28"
vmlist28.SeqID = "20"
vmlist28.IsFavorite = "FALSE"
vmlist28.IsClone = "FALSE"
vmlist28.CfgVersion = "8"
vmlist28.State = "broken"
vmlist28.UUID = "56 4d 85 9f af 9e 91 22-67 db ac e9 66 04 25 3a"
vmlist30.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node10.vmwarevm/maas-node03.vmx"
vmlist30.DisplayName = "maas-node20-65 / fx36c"
vmlist30.ParentID = "54"
vmlist30.ItemID = "30"
vmlist30.SeqID = "12"
vmlist30.IsFavorite = "FALSE"
vmlist30.IsClone = "FALSE"
vmlist30.CfgVersion = "8"
vmlist30.State = "broken"
vmlist30.UUID = "56 4d 32 70 c6 ff d6 4c-4a c6 0a aa 73 c3 34 65"
vmlist31.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node11.vmwarevm/maas-node03.vmx"
vmlist31.DisplayName = "maas-node09-7D"
vmlist31.ParentID = "54"
vmlist31.ItemID = "31"
vmlist31.SeqID = "2"
vmlist31.IsFavorite = "FALSE"
vmlist31.IsClone = "FALSE"
vmlist31.CfgVersion = "8"
vmlist31.State = "broken"
vmlist31.UUID = "56 4d 51 49 75 7a dd 76-d7 5a 48 03 e3 d4 e6 7d"
vmlist32.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node12.vmwarevm/maas-node03.vmx"
vmlist32.DisplayName = "maas-node15-5B-96"
vmlist32.ParentID = "54"
vmlist32.ItemID = "32"
vmlist32.SeqID = "8"
vmlist32.IsFavorite = "FALSE"
vmlist32.IsClone = "FALSE"
vmlist32.CfgVersion = "8"
vmlist32.State = "broken"
vmlist32.UUID = "56 4d b8 42 2b 42 f1 a5-26 f2 92 fb e0 ef 5b 96"
vmlist33.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node13.vmwarevm/maas-node03.vmx"
vmlist33.DisplayName = "maas-node12-12"
vmlist33.ParentID = "54"
vmlist33.ItemID = "33"
vmlist33.SeqID = "5"
vmlist33.IsFavorite = "FALSE"
vmlist33.IsClone = "FALSE"
vmlist33.CfgVersion = "8"
vmlist33.State = "broken"
vmlist33.UUID = "56 4d d5 f0 a5 a8 1a c7-98 b7 9b 75 53 6e b3 12"
vmlist34.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node14.vmwarevm/maas-node03.vmx"
vmlist34.DisplayName = "maas-node08-DA"
vmlist34.ParentID = "54"
vmlist34.ItemID = "34"
vmlist34.SeqID = "1"
vmlist34.IsFavorite = "FALSE"
vmlist34.IsClone = "FALSE"
vmlist34.CfgVersion = "8"
vmlist34.State = "broken"
vmlist34.UUID = "56 4d e8 d4 ac 0b d0 6c-9b c0 c7 0f 16 0c aa da"
vmlist35.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node15.vmwarevm/maas-node03.vmx"
vmlist35.DisplayName = "maas-node18-00 / 6rpyg"
vmlist35.ParentID = "54"
vmlist35.ItemID = "35"
vmlist35.SeqID = "10"
vmlist35.IsFavorite = "FALSE"
vmlist35.IsClone = "FALSE"
vmlist35.CfgVersion = "8"
vmlist35.State = "broken"
vmlist35.UUID = "56 4d 2a 1b 12 fb 91 fa-c9 55 13 c3 65 6b cd 00"
vmlist36.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node16.vmwarevm/maas-node03.vmx"
vmlist36.DisplayName = "maas-node10-AC / nf4xd"
vmlist36.ParentID = "54"
vmlist36.ItemID = "36"
vmlist36.SeqID = "3"
vmlist36.IsFavorite = "FALSE"
vmlist36.IsClone = "FALSE"
vmlist36.CfgVersion = "8"
vmlist36.State = "broken"
vmlist36.UUID = "56 4d e0 86 b6 2f 8e 2e-51 31 69 9a 54 59 b8 ac"
vmlist37.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node17.vmwarevm/maas-node03.vmx"
vmlist37.DisplayName = "maas-node26-62"
vmlist37.ParentID = "54"
vmlist37.ItemID = "37"
vmlist37.SeqID = "18"
vmlist37.IsFavorite = "FALSE"
vmlist37.IsClone = "FALSE"
vmlist37.CfgVersion = "8"
vmlist37.State = "broken"
vmlist37.UUID = "56 4d a9 24 d4 f2 e4 27-6f 08 d7 af 0e ca b5 62"
vmlist38.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node18.vmwarevm/maas-node03.vmx"
vmlist38.DisplayName = "maas-node14-A8 / md6xe"
vmlist38.ParentID = "54"
vmlist38.ItemID = "38"
vmlist38.SeqID = "7"
vmlist38.IsFavorite = "FALSE"
vmlist38.IsClone = "FALSE"
vmlist38.CfgVersion = "8"
vmlist38.State = "broken"
vmlist38.UUID = "56 4d fd 8a fb 68 7d 3d-76 a6 d6 bb c7 31 fb a8"
vmlist39.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node19.vmwarevm/maas-node03.vmx"
vmlist39.DisplayName = "maas-node19-F5"
vmlist39.ParentID = "54"
vmlist39.ItemID = "39"
vmlist39.SeqID = "11"
vmlist39.IsFavorite = "FALSE"
vmlist39.IsClone = "FALSE"
vmlist39.CfgVersion = "8"
vmlist39.State = "broken"
vmlist39.UUID = "56 4d 08 79 36 4d b2 ee-5e 06 d6 d8 e5 11 15 f5"
vmlist40.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node20.vmwarevm/maas-node03.vmx"
vmlist40.DisplayName = "maas-node23-78"
vmlist40.ParentID = "54"
vmlist40.ItemID = "40"
vmlist40.SeqID = "15"
vmlist40.IsFavorite = "FALSE"
vmlist40.IsClone = "FALSE"
vmlist40.CfgVersion = "8"
vmlist40.State = "broken"
vmlist40.UUID = "56 4d 95 69 b3 ce 5a 81-8c 63 69 d3 e5 0a 86 78"
vmlist41.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node21.vmwarevm/maas-node03.vmx"
vmlist41.DisplayName = "maas-node24-FF"
vmlist41.ParentID = "54"
vmlist41.ItemID = "41"
vmlist41.SeqID = "16"
vmlist41.IsFavorite = "FALSE"
vmlist41.IsClone = "FALSE"
vmlist41.CfgVersion = "8"
vmlist41.State = "broken"
vmlist41.UUID = "56 4d 48 a7 d6 bf 40 44-f1 55 1b b6 22 8c 2b ff"
vmlist42.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node22.vmwarevm/maas-node03.vmx"
vmlist42.DisplayName = "maas-node21-2A"
vmlist42.ParentID = "54"
vmlist42.ItemID = "42"
vmlist42.SeqID = "13"
vmlist42.IsFavorite = "FALSE"
vmlist42.IsClone = "FALSE"
vmlist42.CfgVersion = "8"
vmlist42.State = "broken"
vmlist42.UUID = "56 4d 55 74 c5 f2 f0 0f-75 ae ab ed 3a 02 74 2a"
vmlist43.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node23.vmwarevm/maas-node03.vmx"
vmlist43.DisplayName = "maas-node11-73"
vmlist43.ParentID = "54"
vmlist43.ItemID = "43"
vmlist43.SeqID = "4"
vmlist43.IsFavorite = "FALSE"
vmlist43.IsClone = "FALSE"
vmlist43.CfgVersion = "8"
vmlist43.State = "broken"
vmlist43.UUID = "56 4d 45 98 77 89 2a 53-f0 50 15 81 be 86 0c 73"
vmlist44.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node24.vmwarevm/maas-node03.vmx"
vmlist44.DisplayName = "maas-node25-A2"
vmlist44.ParentID = "54"
vmlist44.ItemID = "44"
vmlist44.SeqID = "17"
vmlist44.IsFavorite = "FALSE"
vmlist44.IsClone = "FALSE"
vmlist44.CfgVersion = "8"
vmlist44.State = "broken"
vmlist44.UUID = "56 4d 89 7d d6 c6 98 a4-e1 60 dc a3 2d 04 82 a2"
vmlist45.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node25.vmwarevm/maas-node03.vmx"
vmlist45.DisplayName = "maas-node27-39"
vmlist45.ParentID = "54"
vmlist45.ItemID = "45"
vmlist45.SeqID = "19"
vmlist45.IsFavorite = "FALSE"
vmlist45.IsClone = "FALSE"
vmlist45.CfgVersion = "8"
vmlist45.State = "broken"
vmlist45.UUID = "56 4d d4 64 05 fe 52 64-ce 3e 29 28 36 cd 02 39"
vmlist46.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node26.vmwarevm/maas-node03.vmx"
vmlist46.DisplayName = "maas-node13-60 / y43pw"
vmlist46.ParentID = "54"
vmlist46.ItemID = "46"
vmlist46.SeqID = "6"
vmlist46.IsFavorite = "FALSE"
vmlist46.IsClone = "FALSE"
vmlist46.CfgVersion = "8"
vmlist46.State = "broken"
vmlist46.UUID = "56 4d 74 8e 3c d6 8f 99-6c 10 5b a7 4b 07 67 60"
vmlist47.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node27.vmwarevm/maas-node03.vmx"
vmlist47.DisplayName = "maas-node17-6B"
vmlist47.ParentID = "54"
vmlist47.ItemID = "47"
vmlist47.SeqID = "9"
vmlist47.IsFavorite = "FALSE"
vmlist47.IsClone = "FALSE"
vmlist47.CfgVersion = "8"
vmlist47.State = "broken"
vmlist47.UUID = "56 4d 05 ec 55 24 43 3c-77 06 1f d4 39 51 45 6b"
vmlist48.config = "/Volumes/XRAID5/VIRTUAL_DISKS/maas-node28.vmwarevm/maas-node03.vmx"
vmlist48.DisplayName = "maas-node22-AA"
vmlist48.ParentID = "54"
vmlist48.ItemID = "48"
vmlist48.SeqID = "14"
vmlist48.IsFavorite = "FALSE"
vmlist48.IsClone = "FALSE"
vmlist48.CfgVersion = "8"
vmlist48.State = "broken"
vmlist48.UUID = "56 4d 0b f8 2f 87 41 d9-2b e5 68 7c fc e2 1a aa"
vmlist50.config = "/Volumes/XRAID5/VIRTUAL_DISKS/Ubuntu-MAAS02.vmwarevm/Ubuntu-MAAS02.vmx"
vmlist50.DisplayName = "Ubuntu-MAAS02 (quantal)"
vmlist50.ParentID = "54"
vmlist50.ItemID = "50"
vmlist50.SeqID = "0"
vmlist50.IsFavorite = "FALSE"
vmlist50.IsClone = "FALSE"
vmlist50.CfgVersion = "8"
vmlist50.State = "broken"
vmlist50.UUID = "56 4d 0f 66 49 ad f7 8b-34 93 56 c4 d1 b9 f8 44"
vmlist51.config = "/Volumes/POO/Virtual Machines.localized/Windows 2000 Server.vmwarevm/Windows 2000 Server.vmx"
vmlist51.DisplayName = "Windows 2000 Server"
vmlist51.ParentID = "0"
vmlist51.ItemID = "51"
vmlist51.SeqID = "1"
vmlist51.IsFavorite = "FALSE"
vmlist51.IsClone = "FALSE"
vmlist51.CfgVersion = "8"
vmlist51.State = "normal"
vmlist51.UUID = "56 4d 77 f0 df 92 07 15-8c 81 a2 8f c0 60 dd 8a"
vmlist52.config = "/Volumes/POO/Virtual Machines.localized/Windows 7.vmwarevm/Windows 7.vmx"
vmlist52.DisplayName = "Windows 7"
vmlist52.ParentID = "0"
vmlist52.ItemID = "52"
vmlist52.SeqID = "4"
vmlist52.IsFavorite = "FALSE"
vmlist52.IsClone = "FALSE"
vmlist52.CfgVersion = "8"
vmlist52.State = "normal"
vmlist52.UUID = "56 4d d3 76 9e 01 c5 1f-b3 c1 76 56 56 e6 96 29"
vmlist53.config = "/Volumes/XRAID5/VIRTUAL_DISKS/Ubuntu-MAAS01.vmwarevm/Ubuntu-MAAS01.vmx"
vmlist53.DisplayName = "Ubuntu-MAAS01 (precise)"
vmlist53.ParentID = "23"
vmlist53.ItemID = "53"
vmlist53.SeqID = "0"
vmlist53.IsFavorite = "FALSE"
vmlist53.IsClone = "FALSE"
vmlist53.CfgVersion = "8"
vmlist53.State = "broken"
vmlist53.UUID = "56 4d 7f c6 68 16 74 44-14 92 1c a0 46 69 47 5a"
vmlist54.config = "folder54"
vmlist54.Type = "2"
vmlist54.DisplayName = "MAAS - inactive"
vmlist54.ParentID = "23"
vmlist54.ItemID = "54"
vmlist54.SeqID = "7"
vmlist54.IsFavorite = "FALSE"
vmlist54.UUID = "folder:52 07 93 4e c2 af 7f c3-9e f3 08 86 61 a7 99 dc"
vmlist54.Expanded = "TRUE"
vmlist55.config = "/Volumes/POO/Virtual Machines.localized/GACDTL02KS230M.ITServices.sbc.com/GACDTL02KS230M.ITServices.sbc.com.vmx"
vmlist55.DisplayName = "GACDTL02KS230M.ITServices.sbc.com"
vmlist55.ParentID = "0"
vmlist55.ItemID = "55"
vmlist55.SeqID = "7"
vmlist55.IsFavorite = "FALSE"
vmlist55.IsClone = "FALSE"
vmlist55.CfgVersion = "8"
vmlist55.State = "normal"
vmlist55.UUID = "56 4d 95 e8 fd 02 40 8c-f7 4b b4 b7 1d 35 3c b2"
vmlist56.config = "/Volumes/POO/Virtual Machines.localized/VMs/Ubuntu.vmwarevm/Ubuntu.vmx"
vmlist56.DisplayName = "SPORE"
vmlist56.ParentID = "0"
vmlist56.ItemID = "56"
vmlist56.SeqID = "8"
vmlist56.IsFavorite = "FALSE"
vmlist56.IsClone = "FALSE"
vmlist56.CfgVersion = "8"
vmlist56.State = "normal"
vmlist56.UUID = "56 4d ca 72 43 d5 ae 62-cc 7d da 86 4b 0e ce 26"
vmlist2.config = ""
vmlist3.config = ""
vmlist5.config = ""
vmlist6.config = ""
vmlist7.config = ""
vmlist8.config = ""
vmlist9.config = ""
vmlist10.config = ""
vmlist12.config = ""
vmlist13.config = ""
vmlist14.config = ""
vmlist15.config = ""
vmlist16.config = ""
vmlist17.config = ""
vmlist19.config = ""
vmlist27.config = ""
vmlist29.config = ""
vmlist49.config = ""
