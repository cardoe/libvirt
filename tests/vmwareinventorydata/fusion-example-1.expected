/Volumes/POO/Virtual Machines.localized/MS XP.vmwarevm/MS XP.vmx
/Volumes/POO/Virtual Machines.localized/VMs/ATTM CentOS 64-bit.vmwarevm/ATTM CentOS 64-bit.vmx
/Volumes/POO/Virtual Machines.localized/Windows Vista.vmwarevm/Windows Vista.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node01.vmwarevm/maas-node01.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node03.vmwarevm/maas-node03.vmx
/Volumes/POO/Virtual Machines.localized/VMs/nagiosxi-64/nagiosxi-64.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node04.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node05.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node06.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node07.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node08.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node10.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node11.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node12.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node13.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node14.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node15.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node16.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node17.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node18.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node19.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node20.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node21.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node22.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node23.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node24.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node25.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node26.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node27.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/maas-node28.vmwarevm/maas-node03.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/Ubuntu-MAAS02.vmwarevm/Ubuntu-MAAS02.vmx
/Volumes/POO/Virtual Machines.localized/Windows 2000 Server.vmwarevm/Windows 2000 Server.vmx
/Volumes/POO/Virtual Machines.localized/Windows 7.vmwarevm/Windows 7.vmx
/Volumes/XRAID5/VIRTUAL_DISKS/Ubuntu-MAAS01.vmwarevm/Ubuntu-MAAS01.vmx
/Volumes/POO/Virtual Machines.localized/GACDTL02KS230M.ITServices.sbc.com/GACDTL02KS230M.ITServices.sbc.com.vmx
/Volumes/POO/Virtual Machines.localized/VMs/Ubuntu.vmwarevm/Ubuntu.vmx
